Src/PerformanceCollector/PerformanceCollector/Perf.csproj

```diff
@@ -44,6 +44,7 @@
     <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="4.5.0" />
     <!--Framework References-->
     <Reference Include="System.Runtime.Caching" />
+    <Reference Include="System.Web" />
```

Comment:
I don't remember whether I asked before. Can this be implemented as some sort of quick pulse extensibility point to eliminate this reference?

Reply:
perhaps a sort of a callback?

Reply:
That would be a design change - not in this iteration